### PR TITLE
Add link to core API framework documentation

### PR DESCRIPTION
**Pull request by pscadding**
Reworded some of the sentences, and added a link to the core API section on frameworks.

### CODE REVIEW

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/89c510b525363467ba632a29a7b1e13caf73f125/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/b48e88720235847cac911eeb3291ec0c29099949/index.html)

File: docs/en/toolkit/frameworks.md [hunk 1]
```diff
@@ -7,10 +7,12 @@ lang: en
 
 # Toolkit Frameworks
 
-Toolkit frameworks are collections of code that can be used across apps and engines. The Shotgun Utilities and QT Widgets frameworks are especially useful in app development.
+Toolkit frameworks are collections of code that can be used across apps and engines. You can read more about the frameworks in general [in our core API documentation](https://developer.shotgunsoftware.com/tk-core/platform.html#frameworks).
 
-The **Shotgun Utilities Framework**, or `tk-framework-shotgunutils`, is a collection of Shotgun-related helpers and utilities that make App development easier. It includes a Shotgun (MVC) Model, which makes it easy to connect a Shotgun query with a standard QT View, as well as classes for managing settings, background Shotgun processing, and globals such as schema, statuses, and icons.
+We also provide some pre made frameworks which you can use in your custom apps or app hooks.
```
Comment (tannazie) [inline docs/en/toolkit/frameworks.md]: I'd make three small changes to this sentence:

We provide some pre-made frameworks, which you can use in your custom apps or app hooks.

(remove also, add hyphen, add comma)

File: docs/en/toolkit/frameworks.md [hunk 2]
```diff
@@ -7,10 +7,12 @@ lang: en
 
 # Toolkit Frameworks
 
-Toolkit frameworks are collections of code that can be used across apps and engines. The Shotgun Utilities and QT Widgets frameworks are especially useful in app development.
```
Comment (tannazie) [inline docs/en/toolkit/frameworks.md]: I think I added the "The Shotgun Utilities and QT Widgets frameworks are especially useful in app development." sentence, and my reasoning was that this is not an exhaustive list of *all* FWs that we provide, and *not* mentioning anything about that can be deceiving. I wanted to make mention of the fact that we're calling out two specific FWs here. I'll leave it up to you, but I'd consider adding this sentence back in after the "We also provide some... " sentence below.

Comment (pscadding) [reply docs/en/toolkit/frameworks.md]: Thats fair, but do we provide public documentation for the other frameworks? Are they intended to be used by the public?

Comment (pscadding) [reply docs/en/toolkit/frameworks.md]: How about "We provide a public API for the following frameworks which can be used in app development:"

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/f927fff6a616607dc74e19c68be18008d8a02b65/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/4c4e91318c2d95994bebcf86bfab7e9de7c27ece/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/16578c52408653642c8676add67b3b264a221558/index.html)

File: docs/en/toolkit/frameworks.md (hunk 2 above)
Comment (tannazie) [reply docs/en/toolkit/frameworks.md]: I dig it! Be sure to add a comma before "which".

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/72b890757ccbb1051535ca5fdf58fd58e44a5ced/index.html)

**Pull request merged**